pages/index.jsx

```diff
@@ -70,6 +71,7 @@ const Home = () => {
             <HeroGrid content={content} />
             <About />
             <Menu content={menuContent} />
+            <NewFooter />
```

Comment:
So it wouldn't go here with the rest of the homepage components. Notice how the new footer disappears once you go to a different page.